src/jackdaw/serdes/avro.clj

```diff
@@ -149,7 +149,8 @@
 (defn validate-clj! [this x path expected-type]
   (when-not (match-clj? this x)
     (throw (ex-info (serialization-error-msg x expected-type)
-                    {:path path, :data x}))))
+                    {:path path, :data x}
```

Comment:
shouldn't we use `:clj-data` instead of `data` for consistency with the other raised exceptions?

Reply:
sure, not changed by this pr but no harm in aligning

Reply:
Just realised it's a breaking change 🤔 What do others think about it?

Reply:
Yeah it's changing the behaviour for sure.